src/ProjectList/ProjectActions.tsx

```diff
@@ -19,6 +20,13 @@ export const ProjectActions = (props: ProjectActionsProps) => {
         const { target, title } = action;
         return <Action.OpenInBrowser key={index} title={title} url={target} />;
       })}
+
+      <Action.Open
+        title="Open in Editor"
+        icon={Icon.Code}
+        application={editor}
+        target={`${project_root_path}/${name.toLowerCase()}`}
+      />
```

Comment:
This code comes from https://github.com/artsy/artsy_staff_raycast_extension/pull/4